apps/nsq_to_http/nsq_to_http.go

```diff
@@ -40,7 +40,7 @@ var (
 	maxInFlight = flag.Int("max-in-flight", 200, "max number of messages to allow in flight")
 
 	numPublishers = flag.Int("n", 100, "number of concurrent publishers")
-	mode          = flag.String("mode", "round-robin", "the upstream request mode options: multicast, round-robin, hostpool")
+	mode          = flag.String("mode", "round-robin", "the upstream request mode options: multicast, round-robin, hostpool, epsilon-greedy")
```

Comment:
Yea, I mentioned this in IRC, but I think it's the _right_ behavior change longer term.  I think in the majority of cases this is what people want.